irc_bot_methods.rb

```diff
@@ -0,0 +1,80 @@
+  irc_send("PRIVMSG #rubymonstas :https://www.tierspital.uzh.ch/pferde/magen-darm/kolikoperationen/")
+  irc_send("PRIVMSG #rubymonstas :If you want to return to the start, enter return")
+  
+end
+
+def handle_message_lameness(message)
+  if @repetition_lameness == 0
+    irc_send("PRIVMSG #rubymonstas :You chose lameness. ")
+    irc_send("PRIVMSG #rubymonstas :This is the definition of lameness by the american college of equine surgeons: ")
+    irc_send("PRIVMSG #rubymonstas :Lameness refers to an abnormality of a horse’s gait or stance. It can be caused by pain, a mechanical problem, or a neurological condition. 'Lameness, most commonly results from pain in the musculoskeletal system (muscles, tendons, ligaments, bones, or joints) leading to abnormal movement at the walk, trot, or canter.' ")
+    irc_send("PRIVMSG #rubymonstas :Would you like to start the lameness diagnostic program or would you prefer to just be given some further information to read up on lameness?")
+  end
+  @chosen_problem = :lameness
+end
+
+def lameness_information_quiz(message)
+  irc_send("PRIVMSG #rubymonstas :There are three different causes for lameness:")
+  irc_send("PRIVMSG #rubymonstas :Pain: due to pain, the horse can alter the load on the injured leg and this can cause irregularity in the gait.")
+  irc_send("PRIVMSG #rubymonstas :Mechanical lameness: This is not caused by pain but rather by mechanical issues. Examples for this are patellar fixation or abnormal gait due to scar tissue. ")
+  irc_send("PRIVMSG #rubymonstas :Neurological lameness: Nerval damage hinders the horse from moving in a normal way.")
+  irc_send("PRIVMSG #rubymonstas :What do you think, which cause is the most common?")
+end
+def lameness_pain(message)
+  irc_send("PRIVMSG #rubymonstas :You are correct, pain is the most common cause of lameness. ")
+  irc_send("PRIVMSG #rubymonstas :The severity of the symptoms in lameness can vary: in severe cases, horses often refrain from bearing any weight on the affected limbs, while in light cases, sometimes a slight head bobbing in trot can be the only visible sign.")
+  
+  
+end
+
+def lameness_wrong_answer(message)
+  irc_send("PRIVMSG #rubymonstas :You are incorrect, this is not the most common cause of lameness.")
+  irc_send("PRIVMSG #rubymonstas :You should give it another try!")
+  lameness_information_quiz(message)
+end
+
+def program_lameness_diagnostic(message)
+  irc_send("PRIVMSG #rubymonstas :Welcome to the lameness diagnostic program.")
+  irc_send("PRIVMSG #rubymonstas :The first step towards any diagnostic in veterinary medicine is taking a thorough anamnesis.")
+  irc_send("PRIVMSG #rubymonstas :Please enter the following general informations about your horse: Name, Type, Age, Usage (Sport/Leisure)")
+end
+
+def program_ld_anamnesis(message)
```

Comment:
In this case you need the argument `message`, but in all the other methods above, you can remove it